app/main.py

```diff
@@ -1,19 +1,10 @@
-lucky_number = 777
-pi = 3.14
-one_is_a_prime_number = False
-name = "Richard"
-my_favourite_films = [
-    "The Shawshank Redemption",
-    "The Lord of the Rings: The Return of the King",
-    "Pulp Fiction",
-    "The Good, the Bad and the Ugly",
-    "The Matrix",
-]
-profile_info = ("michel", "michel@gmail.com", "12345678")
-marks = {
-    "John": 4,
-    "Sergio": 3,
-}
-collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [b, d],
+    "immutable": [a, c]}
+a = 5
+b = ["jakow"]
+c = "Hello, worrld"
+d = "orange"
+mutable: [b, d]
```

Comment:
This line appears to be incorrectly formatted. It seems like an attempt to define a dictionary or list, but it lacks the necessary syntax. If you intend to define a dictionary or list, ensure you use the correct syntax.